Assets/Scripts/GameManager.cs

```diff
@@ -212,6 +212,7 @@ public void UpdateLives(int livesLost)
         if(playerLives < 1)
         {
             StartCoroutine(GameEnd());
+
```

Comment:
@elijahscherz ok I added a space here while trying to figure something out... thx github